toolchain/parser/parse_node_kind.def

```diff
@@ -208,12 +208,12 @@ CARBON_PARSE_NODE_KIND_CHILD_COUNT(WhileConditionStart, 0)
 CARBON_PARSE_NODE_KIND_BRACKET(WhileCondition, WhileConditionStart)
 CARBON_PARSE_NODE_KIND_CHILD_COUNT(WhileStatement, 2)
 
-// Parenthesized expressions:
+// Parenthesized expressions, such as `(2)`:
 //   ParenExpressionOrTupleLiteralStart
 //   _external_: expression
 // ParenExpression
 
```

Comment:
```suggestion
//
```

As commented [here](https://github.com/carbon-language/carbon-lang/pull/2992#discussion_r1267113629), let's replace this blank line with an empty comment line to make it clear these comments are connected.